From: Caleb Connolly <caleb.connolly@linaro.org>
Date: Tue, 30 Nov 2021 22:11:39 +0000
Subject: arm64: dts: qcom: sdm845-axolotl: enable fuel gauge

Enable the fuel gauge and configure the associated charger and battery.

Signed-off-by: Caleb Connolly <caleb.connolly@linaro.org>
---
 arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
index fae3234..9496856 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
+++ b/arch/arm64/boot/dts/qcom/sdm845-shift-axolotl.dts
@@ -532,6 +532,12 @@ &pmi8998_charger {
 	status = "okay";
 };
 
+&pmi8998_fg {
+	status = "okay";
+	monitored-battery = <&battery>;
+	power-supplies = <&pmi8998_charger>;
+};
+
 &pmi8998_haptics {
 	status = "okay";
 
